### ISSUE TITLE
Silent failure to upgrade after the cli prompted me to

### ISSUE THREAD

**Issue by cschneid** [plandex-ai/plandex#114]
```
$ plandex --help
A new version of Plandex is available: 0.9.1
Current version: 0.9.0
Upgrade to the latest version? (y)es | (n)o> y
```

I get a quick progress bar, then the help docs of plandex as I originally asked. But it didn't actually upgrade anything. I don't see any error message at all. Rerunning causes the same prompt and failed upgrade behavior.

The permissions should allow an update, the binary is owned by my user.

```
$ which plandex
/usr/local/bin/plandex

$ ls -l /usr/local/bin/plandex
-rwxr-xr-x@ 1 cschneid  staff  21123954 Apr 29 12:06 /usr/local/bin/plandex
```

**Comment by cschneid** [comment]
Update, I manually went to update it, and it required me to use sudo to get the new copy over the old one. Maybe it is the parent dir that is breaking the upgrade by requiring root perms to move the new file into it.

```
$ ls -ltr /usr/local/
total 0
drwxr-xr-x   9 root  wheel  288 Jun 10  2022 sessionmanagerplugin
drwxr-xr-x   5 root  wheel  160 Jun 16  2022 share
drwxr-xr-x   3 root  wheel   96 Jun 16  2022 include
drwxr-xr-x   5 root  wheel  160 Jun 29  2022 lib
drwxr-xr-x  22 root  wheel  704 May  8 20:07 bin
```

**Comment by danenania** [comment]
This does sound similar to the permission issue in #97. If that's the case, you should get output telling you to re-run the command with sudo, but we won't find out for sure until the next release since the fix is in 0.9.1.

**Comment by cschneid** [comment]
Should have searched :) It sure looks like my scenario in that ticket, I'll close this one.
